src/set_theory/zfc/basic.lean

```diff
@@ -1095,7 +1092,7 @@ mem_univ.2 $ or.elim (classical.em $ ∃ x, ∀ y, A y ↔ y = x)
  (λ hn, ⟨∅, ext (λ z, coe_empty.symm ▸ ⟨false.rec _, λ ⟨._, ⟨x, rfl, H⟩, zA⟩, hn ⟨x, H⟩⟩)⟩)
 
 /-- Function value -/
-def fval (F A : Class.{u}) : Class.{u} := iota (λ y, to_Set (λ x, F (Set.pair x y)) A)
+def fval (F A : Class.{u}) : Class.{u} := iota $ λ y, Class.has_mem.mem A $ λ x, F (Set.pair x y)
```

Comment:
Hmm, this feels pretty weird. Do we not have syntax to construct a `Class` from a lambda?

Reply:
Perhaps
```suggestion
def fval (F A : Class.{u}) : Class.{u} := iota $ λ y, A ∈ {x | F (Set.pair x y)}
```

Reply:
This isn't able to find the `∈` instance, since the instance searcher can't see through the def-eq `set Set = Class`.